controller/userController.js

```diff
@@ -0,0 +1,48 @@
+const { catchAsync } = require('../middleware/error.js');
+const { isExistedUser } = require('../model/userDao.js');
+const userService = require('../service/userService.js');
+
+const signUp = catchAsync(async (req, res) => {
+  const {
+    email,
+    password,
+    name,
+    phoneNumber,
+    address,
+    gender,
+    birthDate,
+    points = 100000.0,
+  } = req.body;
```

Comment:
포인트 값의 자유도를 위해 환경변수를 활요하면 좋을 것 같습니다.
```suggestion
    gender,
    birthDate,
    points = parseFloat(process.env.POINTS) || 100000.0,
  } = req.body;
```